packages/admin/admin/src/translator/ContentTranslationServiceProvider.tsx

```diff
@@ -2,6 +2,15 @@ import * as React from "react";
 
 import { ContentTranslationServiceContext } from "./ContentTranslationServiceContext";
 
-export const ContentTranslationServiceProvider: React.FunctionComponent<ContentTranslationServiceContext> = ({ children, enabled, translate }) => {
-    return <ContentTranslationServiceContext.Provider value={{ enabled, translate }}>{children}</ContentTranslationServiceContext.Provider>;
+export const ContentTranslationServiceProvider: React.FunctionComponent<ContentTranslationServiceContext> = ({
+    children,
+    enabled,
+    showApplyTranslationDialog = true,
```

Comment:
@thomasdax98 should this really be `true` by default?

Reply:
No, let's make it false for now

Reply:
330e3f21a57684049e9078028df78aba05d5c884